cmd/nvkind/cluster_create.go

```diff
@@ -135,6 +174,19 @@ func runClusterCreate(c *cli.Context, f *ClusterCreateFlags) error {
 		}
 	}
 
+	// Display endpoints against which kubectl is configured, for debug.
```

Comment:
assumption: at this point `kind` has been invoked and it _did_ configure said kubectl context